segfault during onnegotiationneeded

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x8106bb]

goroutine 22628 [running]:
github.com/pion/webrtc/v3.(*Track).Label(0x0, 0x0, 0x0)
	/go/pkg/mod/github.com/pion/webrtc/v3@v3.0.0-beta.4.0.20200908143909-40eb1f2404c0/track.go:73 +0x3b
github.com/pion/webrtc/v3.(*Track).Msid(0x0, 0x0, 0x4)
	/go/pkg/mod/github.com/pion/webrtc/v3@v3.0.0-beta.4.0.20200908143909-40eb1f2404c0/track.go:87 +0x2f
github.com/pion/webrtc/v3.(*PeerConnection).checkNegotiationNeeded(0xc000506340, 0x813501)
	/go/pkg/mod/github.com/pion/webrtc/v3@v3.0.0-beta.4.0.20200908143909-40eb1f2404c0/peerconnection.go:360 +0x833
github.com/pion/webrtc/v3.(*PeerConnection).negotiationNeededOp(0xc000506340)
	/go/pkg/mod/github.com/pion/webrtc/v3@v3.0.0-beta.4.0.20200908143909-40eb1f2404c0/peerconnection.go:303 +0x90
github.com/pion/webrtc/v3.(*operations).start(0xc0005186c0)
	/go/pkg/mod/github.com/pion/webrtc/v3@v3.0.0-beta.4.0.20200908143909-40eb1f2404c0/operations.go:84 +0x61
created by github.com/pion/webrtc/v3.(*operations).Enqueue
	/go/pkg/mod/github.com/pion/webrtc/v3@v3.0.0-beta.4.0.20200908143909-40eb1f2404c0/operations.go:35 +0xcb
```

another related:
```
[2020-10-07 17:23:20.586] [INFO] [2041][server.go][ServeHTTP] => Closing peer
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x14a3ebc]

goroutine 8975 [running]:
github.com/pion/webrtc/v3.(*RTPSender).Track(0x0, 0x0)
	/Users/tvanas/go/pkg/mod/github.com/pion/webrtc/v3@v3.0.0-beta.7/rtpsender.go:80 +0x3c
github.com/pion/webrtc/v3.(*PeerConnection).checkNegotiationNeeded(0xc00035a000, 0x1)
	/Users/tvanas/go/pkg/mod/github.com/pion/webrtc/v3@v3.0.0-beta.7/peerconnection.go:360 +0x84e
github.com/pion/webrtc/v3.(*PeerConnection).negotiationNeededOp(0xc00035a000)
	/Users/tvanas/go/pkg/mod/github.com/pion/webrtc/v3@v3.0.0-beta.7/peerconnection.go:301 +0x90
github.com/pion/webrtc/v3.(*operations).start(0xc00042d7a0)
	/Users/tvanas/go/pkg/mod/github.com/pion/webrtc/v3@v3.0.0-beta.7/operations.go:84 +0x61
created by github.com/pion/webrtc/v3.(*operations).Enqueue
	/Users/tvanas/go/pkg/mod/github.com/pion/webrtc/v3@v3.0.0-beta.7/operations.go:35 +0xcb
```

--- a/peerconnection.go
+++ b/peerconnection.go
@@ -353,7 +353,7 @@ func (pc *PeerConnection) checkNegotiationNeeded() bool { //nolint:gocognit
 			// Step 5.3.1
 			if t.Direction() == RTPTransceiverDirectionSendrecv || t.Direction() == RTPTransceiverDirectionSendonly {
 				descMsid, okMsid := m.Attribute(sdp.AttrKeyMsid)
-				if !okMsid || descMsid != t.Sender().Track().Msid() {
+				if !okMsid || (t.Sender() != nil && descMsid != t.Sender().Track().Msid()) {
 					return true
 				}
 			}

